Source/DafnyDriver.Test/OptionsTest.cs

```diff
@@ -0,0 +1,77 @@
+using System.CommandLine;
+using System.CommandLine.Builder;
+using System.CommandLine.Invocation;
+using System.CommandLine.Parsing;
+using System.Diagnostics;
+using System.IO.Pipelines;
+using System.Reactive.Concurrency;
+using System.Reflection;
+using System.Text;
+using Microsoft.Dafny;
+using Microsoft.Extensions.Logging.Abstractions;
+using OmniSharp.Extensions.JsonRpc;
+using OmniSharp.Extensions.JsonRpc.Client;
+using OmniSharp.Extensions.JsonRpc.Serialization;
+using OmniSharp.Extensions.LanguageServer.Protocol.Client.Capabilities;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+using XunitAssertMessages;
+using Command = System.CommandLine.Command;
+
+namespace DafnyDriver.Test;
+
+
+public class OptionsTest {
+  [Fact]
+  public async Task RunFlagsOverride() {
+    await TestCliRunArgs([], options => {
+      Assert.True(options.DafnyVerify);
```

Comment:
I'm not a fan of tests for values in `options.`, since these are implementation details. In the future we could do a refactoring that removes `options.DafnyVerify`, and then what would we do with this test?

Reply:
Implementation details can still be tested by unit tests, and this is the meaning of that.
If we remove `options.DafnyVerify`, we would simply remove that test.

Reply:
I did not know about `--progress Symbol` until your message, that looks like it would make a nice integration test. I can envision creating a test based on that if you prefer.
But first, what makes you uncomfortable with unit tests like this?

Reply:
> But first, what makes you uncomfortable with unit tests like this?

Concretely, I plan on removing `DafnyOptions.DafnyVerify` and this test will make that harder. More generally, for tests against implementation it's hard to say how long it takes they need to be replaced. It'd rather not create work further down the line.

Reply:
`--progress Symbol`  is not producing any difference between the two command lines:

```
> make test-dafny name="rust/datatypes.dfy" action="run -t:rs --progress Symbol --no-verify"
> make test-dafny name="rust/datatypes.dfy" action="run -t:rs --progress Symbol"
```
The two commands output exactly the same thing.
I see that there is a barely used option `options.Get(CommonOptionBag.LogLevelOption);` that I could configure so that if it's set to verbose, it will emit something when entering translation. Would that work for you?

Reply:
Right, `--progress` is not supported by the commands that have not been ported to the new front-end yet, so it's ignored.

If `options.Get(CommonOptionBag.LogLevelOption)` works that's great too.